src/Microsoft.Data/PrimitiveColumn.cs

```diff
@@ -479,14 +460,14 @@ public override BaseColumn Filter<U>(U lower, U upper)
                 return _Filter((T)convertedLower, (T)Convert.ChangeType(upper, typeof(T)));
             }
             else
-                throw new ArgumentException(Strings.MismatchedValueType + typeof(T).ToString(), nameof(U));
+                throw new ArgumentException(string.Format(Strings.MismatchedValueType, typeof(T)), typeof(U).ToString());
         }
 
         public override DataFrame Description()
         {
             DataFrame ret = new DataFrame();
             StringColumn stringColumn = new StringColumn("Description", 0);
-            stringColumn.Append("Length");
+            stringColumn.Append("Length(non null values)");
```

Comment:
For the future - we should probably put these strings into a resource. End-users will see these strings.

Reply:
https://github.com/dotnet/corefxlab/issues/2736